extensions/HamzaMath

```diff
@@ -0,0 +1,178 @@
+                            }
+                        },
+                        group: 'Calculates' // Group for the blocks
+                    },
+                    {
+                        opcode: 'divide',
+                        blockType: Scratch.BlockType.REPORTER,
+                        text: 'Divide: [NUMBERS]',
+                        arguments: {
+                            NUMBERS: {
+                                type: Scratch.ArgumentType.STRING,
+                                defaultValue: '1, 2'
+                            }
+                        },
+                        group: 'Calculates' // Group for the blocks
+                    },
+                    {
+                        opcode: 'modulus',
+                        blockType: Scratch.BlockType.REPORTER,
+                        text: 'Modulus: [NUMBERS]',
+                        arguments: {
+                            NUMBERS: {
+                                type: Scratch.ArgumentType.STRING,
+                                defaultValue: '1, 2'
+                            }
+                        },
+                        group: 'Calculates' // Group for the blocks
+                    },
+                    // Separator between the two groups
+                    {
+                        opcode: 'separator2',
+                        blockType: Scratch.BlockType.LABEL,
+                        text: '--- Convert ---',
+                        group: 'Convert' // Conversion group
+                    },
+                    // Conversion group
+                    {
+                        opcode: 'toPositive',
+                        blockType: Scratch.BlockType.REPORTER,
+                        text: 'Convert [NUMBER] to positive',
+                        arguments: {
+                            NUMBER: {
+                                type: Scratch.ArgumentType.NUMBER,
+                                defaultValue: 1
+                            }
+                        },
+                        group: 'Convert' // Conversion group
+                    },
+                    {
+                        opcode: 'toNegative',
+                        blockType: Scratch.BlockType.REPORTER,
+                        text: 'Convert [NUMBER] to negative',
+                        arguments: {
+                            NUMBER: {
+                                type: Scratch.ArgumentType.NUMBER,
+                                defaultValue: 1
+                            }
+                        },
+                        group: 'Convert' // Conversion group
+                    }
+                ]
+            };
+        },
+        calculate: function(args) {
+            try {
+                const result = eval(args.EXPRESSION);
+                return result;
+            } catch (error) {
+                return 'Error'; // Return error message for invalid expression
+            }
```

Comment:
You should **never** be using eval like this. People will be able to run any javascript code, which is dangerous